server/controller/index.js

```diff
@@ -21,6 +21,8 @@ const {
 const { login } = require('./routes/admin');
 const { validateEvent, createEvent } = require('./routes/admin');
 const { validateAttendence, checkUserBooking, signAttendance } = require('./routes/portal');
+const { deleteEvent } = require('./routes/admin/deleteEvent');
```

Comment:
```suggestion
const { deleteEvent } = require('./routes/admin');
```
you exported it to admin/index so, you can require it from index